modles/postModel.js

```diff
@@ -3,26 +3,37 @@ function postArticleMdl(req) {
   const { article_id } = req.params;
   return db
     .query(
-      "INSERT INTO comments (body, author, article_id, votes)VALUES($1,$2,$3,$4) RETURNING *;",
-      [req.body[0]["body"], req.body[0]["username"], article_id, 0]
+      "INSERT INTO comments (body, author, article_id, votes,created_at)VALUES($1,$2,$3,$4,$5) RETURNING *;",
+      [
+        req.body[0]["body"],
+        req.body[0]["username"],
+        article_id,
+        0,
+        "2023-01-11T00:00:00.000Z",
+      ]
     )
     .then((result) => {
-      return result.rows;
-    })
-    .catch(() => {
-      return Promise.reject({ status: 404, msg: "not found" });
+      if (result.rows === 0)
+        return Promise.reject({ status: 404, msg: "not found" });
+      else return result.rows;
     });
 }
-function patcharticle_idMdl(req) {
+
+function updateVotes(req) {
   const { article_id } = req.params;
-  console.log(req);
+  const { inc_votes } = req.body;
   return db
-    .query("UPDATE articles SET votes += $1 RETURNING *;", [article_id])
+    .query(
+      `UPDATE articles SET votes = votes + ${inc_votes} WHERE article_id = ${article_id} RETURNING *;`
+    )
```

Comment:
here we are passing in article_id and inc_votes which leaves this query open to SQL injection, how could we pass these values in to avoid this?